Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -65,7 +65,8 @@ - (CGRect)sourceRectInView:(UIView *)sourceView
   NSArray<NSString *> *buttons = [RCTConvert NSStringArray:options[@"options"]];
   NSInteger destructiveButtonIndex = options[@"destructiveButtonIndex"] ? [RCTConvert NSInteger:options[@"destructiveButtonIndex"]] : -1;
   NSInteger cancelButtonIndex = options[@"cancelButtonIndex"] ? [RCTConvert NSInteger:options[@"cancelButtonIndex"]] : -1;
-
+  UIColor *tintColor = options[@"tintColor"] ? [RCTConvert UIColor:options[@"tintColor"]] : nil;
```

Comment:
This change causes an error to be thrown.
`[__NSCFNumber set]: unrecognized selector sent to instance 0xb000000ff0080003`
`[__NSCFNumber set]: unrecognized selector sent to instance 0xb000000ff0080003' was thrown while invoking showActionSheetWithOptions on target RCTActionSheetManager`